### ISSUE TITLE
Getting TimeoutException when initiating chrome browser in grid nodes

### ISSUE THREAD

**Issue by SandeepK-GitHub** [SeleniumHQ/selenium#6805]
Getting TimeoutException when initiating chrome browser in grid nodes

Hi i am getting below exception when chrome browser is initiated in grid nodes. Here my environment setup is like below. First job1 test will run in jenkinns and job2 test will execute once api is done as a downstream. Here getting issue in jenkinns when job2  is triggered but i don't see any issue when i clicked on build now manually for job2. Issue with only when downstream job is triggered.

Jan 02, 2019 7:26:45 AM org.openqa.selenium.remote.ProtocolHandshake createSession
INFO: Detected dialect: OSS
Jan 02, 2019 7:26:45 AM org.openqa.selenium.remote.ProtocolHandshake createSession
INFO: Detected dialect: OSS
Jan 02, 2019 7:26:46 AM org.openqa.selenium.remote.ProtocolHandshake createSession
INFO: Detected dialect: OSS
Jan 02, 2019 7:26:46 AM org.openqa.selenium.remote.ProtocolHandshake createSession
INFO: Detected dialect: OSS
Jan 02, 2019 7:26:46 AM org.openqa.selenium.remote.ProtocolHandshake createSession
INFO: Detected dialect: OSS
org.openqa.selenium.TimeoutException: timeout
  (Session info: chrome=71.0.3578.98)
  (Driver info: chromedriver=2.37.543627 (63642262d9fb93fb4ab52398be4286d844092a5e),platform=Windows NT 6.1.7601 SP1 x86_64) (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 300.04 seconds
Build info: version: '3.4.0', revision: 'unknown', time: 'unknown'
System info: host: 'S-S3-NODE08', ip: '10.43.39.163', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_191'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities [{applicationCacheEnabled=false, rotatable=false, mobileEmulationEnabled=false, networkConnectionEnabled=false, chrome={chromedriverVersion=2.37.543627 (63642262d9fb93fb4ab52398be4286d844092a5e), userDataDir=C:\Users\griduser\AppData\Local\Temp\scoped_dir4520_27923}, takesHeapSnapshot=true, pageLoadStrategy=normal, databaseEnabled=false, handlesAlerts=true, hasTouchScreen=false, version=71.0.3578.98, platform=XP, browserConnectionEnabled=false, nativeEvents=true, acceptSslCerts=false, acceptInsecureCerts=false, locationContextEnabled=true, webStorageEnabled=true, browserName=chrome, takesScreenshot=true, javascriptEnabled=true, cssSelectorsEnabled=true, setWindowRect=true, unexpectedAlertBehaviour=}]
Session ID: 80af860a2f38cb4db1533caa9ea93e5e
Command duration or timeout: 0 milliseconds
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)

Please help on this

**Comment by diemol** [comment]
Hi @SandeepK-GitHub,

Could you please upgrade to the latest Grid version (3.141.59)? If the error still persists, please create a new issue and fill out the complete template.
